dlpy/tests/test_model.py

```diff
@@ -617,6 +618,18 @@ def test_model23(self):
 
         model1.deploy(self.data_dir, output_format='onnx')
 
+    def test_model24(self):
```

Comment:
I was wondering if you can add another onnx example for tests. I can't evaluate the coverage but two would cover more than one.

Reply:
Added additional test cases